newsletters/2022-07.md

```diff
@@ -0,0 +1,65 @@
+---
+tags: obf
+---
+
+_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/2022-07.md)._
+
+_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._
+
+# OBF Newsletter July/August 2022
+
+BOSC 2022 was amazing; BOSC 2023 will be part of ISMB/ECCB in Lyon, France + online. Our next event fellowship application deadline is Oct 1. The Nextflow community is having a get-together, GA4GH wants your thoughts on their Code of Conduct, and educators are invited to use the cloud to teach genomics.
+
+In longer form: 
+
+## OBF News
+
+### BOSC News
+
+**[BOSC 2022](http://open-bio.org/events/bosc/)** took place in July as a COSI (Community of Special Interest) of [ISMB 2022](https://www.iscb.org/ismb2022), which was held as a hybrid event, in-person in Madison, WI (USA) and online. Over the two days of the BOSC COSI, approximately 200 people participated in [BOSC sessions](https://www.open-bio.org/events/bosc-2022/bosc-2022-schedule/), which included 28 talks chosen from submitted abstracts, 46 posters, and a [panel discussion, “Building and Sustaining Inclusive Open Science Communities”](https://www.open-bio.org/events/bosc-2022/bosc-2022-panel/). BOSC 2022 included [joint keynotes](https://www.open-bio.org/events/bosc-2022/bosc-2022-keynotes/) with two other COSIs. Jason Williams gave a BOSC / Education COSI keynote entitled "Riding the bicycle: Including all scientists on a path to excellence". A joint session with Bio-Ontologies featured a keynote by Melissa Haendel, “The open data highway: turbo-boosting translational traffic with ontologies.”
+
+A report about BOSC 2022 is in progress, and will appear as part of the [BOSC collection on F1000](https://f1000research.com/collections/bosc). All BOSC presenters are encouraged to submit their presentation slides to the collection by September 30, 2022.
```

Comment:
their presentation slides and/or posters

Reply:
done